.travis.yml

```diff
@@ -48,6 +48,9 @@ jobs:
     - stage:
       <<: *node-prod
       env: COMMAND=nsp-check
+    - stage:
+      <<: *node-prod
+      env: COMMAND=prettier-ci
```

Comment:
Ugh. This is a problem because it takes a long time (16 secs for me) and this stage is run *sequentially*, not [in parallel](https://docs.travis-ci.com/user/build-stages/). We should run it in parallel but I don't think there is an easy way to fix that. Unless you have an idea, don't worry about it in this patch. I filed this bug: https://github.com/mozilla/addons-frontend/issues/5366